Library/Homebrew/cask/dsl/caveats.rb

```diff
@@ -114,15 +114,16 @@ def eval_caveats(&block)
             #{@cask} requires Java. You can install the latest version with:
               brew install --cask adoptopenjdk
```

Comment:
I wonder if this could/why it's not recommending `openjdk` formulae.

Reply:
Thank you for your questions, @MikeMcQuaid! 

As @reitermarkus [stated](https://github.com/Homebrew/homebrew-cask/issues/16383#issuecomment-726548725) in response to this question:

> `depends_on formula: 'openjdk'` does not install it globally, i.e. link it to `/Library/Java/JavaVirtualMachines` which most GUI apps probably depend on.

Reply:
Makes sense, thanks 👍🏻